lib/helpers/make-replacements.js

```diff
@@ -30,10 +30,17 @@ module.exports = function makeReplacements(contents, from, to) {
   from.forEach((item, i) => {
 
     //Get replacement value
-    const replacement = getReplacement(to, isArray, i);
+    let replacement = getReplacement(to, isArray, i);
     if (replacement === null) {
       return;
     }
+    if (typeof replacement === 'function') {
+      const original = replacement;
+      replacement = function(...args) {
+        args.push(file);
+        return original.apply(null, args);
```

Comment:
We should be able to use the cleaner ES6 syntax `return original(...args, file);` here instead of pushing to the array and using apply.

Reply:
Yeah. That's would be cleanup. Will update.